app/code/Magento/Eav/Model/Api/SearchCriteria/CollectionProcessor/FilterProcessor.php

```diff
@@ -64,20 +64,55 @@ private function addFilterGroupToCollection(
         AbstractDb $collection
     ) {
         $fields = [];
+        $customFilters = [];
```

Comment:
can we use like this `$fields = $customFilters = [];`, right?

Reply:
I would recommend to keep one statement per line, for better readability. Code is easier to read when the logic flows from top to bottom.

Reply:
> can we use like this `$fields = $customFilters = [];`, right?

Sorry, but I cannot agree with you